specs/deneb/p2p-interface.md

```diff
@@ -295,7 +296,7 @@ Before consuming the next response chunk, the response reader SHOULD verify the
 
 No more than `MAX_REQUEST_BLOB_SIDECARS` may be requested at a time.
 
-`BlobSidecarsByRoot` is primarily used to recover recent blobs (e.g. when receiving a block with a transaction whose corresponding blob is missing).
+`BlobSidecarsByRoot` is primarily used to recover recent blobs (e.g. when receiving a block with a transaction whose corresponding blob is missing). In addition, it is possible to retrieve recent blobs by querying the Execution Layer via `engine_getBlobsV1` engine API method.
```

Comment:
Does this sentence mean that an external peer can trigger the node into doing an EL query?

Reply:
I see what you mean. Probably worth better clarifying, but the context here is on the "consumer" of the byRoot, so my intent was to mention this new method that can be used  as alternative (or in combination) to p2p RPC for recovering recent blobs.
EL query triggered by RCP seems to me an edge case, theoretically possible, I'd leave it as an implementation detail.